src/studio/nuxeo-studio/tutorials/how-to-customize-theme-studio.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'HOWTO: Create Themes with Nuxeo Studio'
```

Comment:
As we are already under Nuxeo Studio section, we probably don't need it in title.
```suggestion
title: 'HOWTO: Create Themes'
```